# Standard Header
std_msgs/Header header

# Array of `object_stamped` topics
simple_zed2_wrapper/Object[] objects
